### PR TITLE
#166674063: Save user test score

### PR DESCRIPTION
**Pull request by Tyak99**
### What does this PR do?

* Save user test

### Description of Task to be completed?

* save test score to the database
* update cummulative score in profile
* document feature

### Any background context you want to provide?

A user's test gets saved to the database, and the cumulative in the database gets updated if the user passes the bench score. If the user has passed the taken test initially, then the cumulative score won't be updated.

### What are the relevant Github issues?

N/A

### Questions:

N/A

### CODE REVIEW

File: server/controller/test.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
+import Sequelize from 'sequelize';
 import models from '../models';
 
-const { Test, Question } = models;
+const { Op } = Sequelize;
```
Comment (fortunee) [inline server/controller/test.js]: I think this is more readable `const { Op as Operator } = Sequelize;`

**Review by fortunee — CHANGES_REQUESTED** [review]
Please review comments and refactor accordingly

File: server/controller/test.js (hunk 1 above)
Comment (Tyak99) [reply server/controller/test.js]: Alright. I will make the change

**Review by fortunee — APPROVED** [review]
LGTM

**Pull request merged**